cli/src/auth-server.test.ts

```diff
@@ -107,9 +107,7 @@ describe('handler', () => {
     httpHandler(req, res);
 
     expect(res.writeHead).toHaveBeenCalledWith(500);
-    expect(res.end).toHaveBeenCalledWith(
-      'Something went wrong: error:04099079:rsa routines:RSA_padding_check_PKCS1_OAEP_mgf1:oaep decoding error'
-    );
+    expect(res.end).toHaveBeenCalledWith('Something went wrong: error:02000079:rsa routines::oaep decoding error');
```

Comment:
Non blocking suggestion (I have not run this code).

```suggestion
    expect(res.end).toHaveBeenCalledWith(expect.stringContaining('Something went wrong:'))
    expect(res.end).toHaveBeenCalledWith(expect.stringContaining('decoding error'))
```